pandas/io/pytables.py

```diff
@@ -1735,38 +1707,12 @@ def _write_to_group(
         errors: str = "strict",
         track_times: bool = True,
     ):
-        group = self.get_node(key)
-
-        # we make this assertion for mypy; the get_node call will already
-        #  have raised if this is incorrect
-        assert self._handle is not None
-
-        # remove the node if we are not appending
-        if group is not None and not append:
-            self._handle.remove_node(group, recursive=True)
```

Comment:
what happened to this?

Reply:
I put it in the method ``_identify_group`` below.